privacy_bot/analysis/classifier.py

```diff
@@ -1,3 +1,17 @@
+"""
+Privacy Bot - Text Classifier
+
+Usage:
+    classifier.py -p TRUE_POSITIVES  -n TRUE_NEGATIVES -e ENTITY
+
+Options:
+    -p --tp TRUE_POSITIVES        Local Path of TRUE_POSITIVES (tar file)
+    -n --tn TRUE_NEGATIVES        Local Path of TRUE_NEGATIVES (tar file)
+    -e --entity ENTITY            Two possible values: text, url
```

Comment:
Since we need one of them at least, we should probably use  an alternative in the `Usage` instead: `(--url | --text)`. Which would give:
```
Usage:
    classifier.py -p TRUE_POSITIVES  -n TRUE_NEGATIVES (--url | --text)
```

Also, you don't really need to specify a flag for the options `-p` and `-n` since they are always present anyway (it's also a matter of taste, I'm just proposing an alternative). Could be something like:
```
Usage:
    classifier.py <positive>  <negative> (--url | --text)
```

(They are now positional arguments)